src/util.py

```diff
@@ -141,14 +141,14 @@ def prepare_dataset(train_data,map_toxic=True):
 
     train_data["text"] = train_data["text"].map(str)
     train_data["original_text"] = train_data["text"]
-    train_data["uppercase"] = train_data["text"].map(percent_uppercase)
+    train_data["uppercase"] = train_data["text"].map(text_modifier.percent_uppercase)

```

Comment:
what's going on here with text_modifier?

Reply:
This function is defined in text_modifier.py. I added the module name here so that I can easily tell which file it comes from..